agent/src/beerocks/monitor/monitor_thread.cpp

```diff
@@ -801,19 +801,13 @@ bool monitor_thread::handle_cmdu_vs_message(Socket &sd, ieee1905_1::CmduMessageR
         }
 
         auto sta_node = mon_db.sta_find(sta_mac);
-        if (sta_node) {
-            mon_db.sta_erase(sta_mac);
-        }
-
-        auto vap_node = mon_db.vap_get_by_id(vap_id);
-        if (vap_node == nullptr) {
-            LOG(ERROR) << "vap_id " << vap_id << " doesn't not exists";
+        if (!sta_node) {
+            LOG(ERROR) << "sta " << sta_mac << " hasn't been found on mon_db";
```

Comment:
I don't understand why do we need that message?
ipv4 is updated externally from the controller now, and `bridge_4addr_mac` from what I see on association handling task is filled only when the station is an extender and it is filled with the bridge mac of the station (a platform which is a station).

It s not critical, but it could be removed. Wasn't it the purpose of this PR and its friends?

One more thing.
The commit message headline is too long "agent: monitor: supply bridge_4addr_mac and ipv4 by START_MONITORING_……REQUEST",
and there is a typo on the commit message: 
"It does not trigger measurements per _**se**_, it supplies missing data"